images/miq-app/docker-assets/appliance-initialize.sh

```diff
@@ -11,8 +11,11 @@ sleep "${APPLICATION_INIT_DELAY}"
 # Prepare initialization environment
 prepare_init_env
 
+# Check Memcached readiness
+check_svc_status ${MEMCACHED_SERVICE_NAME} 11211
+
 # Check DB readiness
-check_db_status
+check_svc_status ${DATABASE_SERVICE_NAME} 5432
```

Comment:
Isn't pg_isready better for checking postgres ? 
In case it does something else  in addition to open a tcp socket than we may want to keep it ?